### PR TITLE
Fix Test 1 wording

### PR DESCRIPTION
**Pull request by ghost**
Fix Test 1 wording so that "elements" doesn't look like part of the code, as per #3260.

### CODE REVIEW

**Comment by awesomeaniruddh** [comment]
Thanks for the PR @leonfeng, but unfortunately there are merge conflicts. Can you please resolve them?

Also, please read this article thorougly: https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Pull-Request-Contribute

Thanks!

**Comment by leonfeng** [comment]
Thanks for the tips, @awesomeaniruddh. Working on it.

**Comment by awesomeaniruddh** [comment]
Also, can you rephrase the test to something along the lines of what I suggested?

**Comment by leonfeng** [comment]
Done. Closing this PR and opening a new one.
